log_transfer/tests/audit_logging.py

```diff
@@ -34,38 +38,58 @@ def log(
     get_time: Callable[[], datetime] = _now,
     ip_address: str = "",
     additional_information: str = "",
+    audit_logger_type: AuditLoggerType = AuditLoggerType.SINGLE_COLUMN_JSON,
 ):
     current_time = get_time()
-    user_id = str(actor.pk) if getattr(actor, "pk", None) else ""
 
-    role = Role.SYSTEM
-    
-    message = {
-        "audit_event": { # See env variable DATE_TIME_PARENT_FIELD if changing this
-            "origin": settings.AUDIT_LOG_ORIGIN,
-            "status": str(status.value),
-            "date_time_epoch": int(current_time.timestamp() * 1000),
-            "date_time": _iso8601_date(current_time), # See env variable DATE_TIME_FIELD if changing this
-            "actor": {
-                "role": str(role.value),
-                "user_id": user_id,
-                "provider": actor_backend
-                if actor_backend
-                else "",
-                "ip_address": ip_address,
-            },
-            "operation": str(operation.value),
-            "additional_information": additional_information,
-            "target": {
-                "id": _get_target_id(target),
-                "type": _get_target_type(target),
+    if audit_logger_type == AuditLoggerType.SINGLE_COLUMN_JSON:
```

Comment:
match-case might work here as well, especially if we have more options in the future. Can be easily refactored at that time as well.

Reply:
See [this comment](https://github.com/City-of-Helsinki/structured-log-transfer/pull/3#discussion_r1203921110).

Reply:
ok